rotkehlchen/exchanges/binance.py

```diff
@@ -589,6 +607,184 @@ def _query_lending_balances(
 
         return balances
 
+    def _api_query_list_paginated_within_time_delta(
+            self,
+            start_ts: Timestamp,
+            end_ts: Timestamp,
+            time_delta: int,
+            api_type: BINANCE_API_TYPE,
+            method: Literal['lending/union/interestHistory'],
+            additional_options: Optional[dict] = None,
+    ) -> list[dict[str, Any]]:
+        """Request via `_api_query_list_within_time_delta` using `current` param for pagination"""
```

Comment:
This docstring is not very useful since I had to look at the code to understand what it was doing. Can you maybe edit it to

```suggestion
        """Query binance lending endpoint for a time range iterating in periods of `time_delta` and using pagination"""
```

or any other info that adds value